src/utils/reclient.js

```diff
@@ -106,17 +110,46 @@ function reclientEnv(config) {
   };
 }
 
-function ensureHelperAuth(config) {
-  const result = childProcess.spawnSync(reclientHelperPath, ['status'], {
-    stdio: 'pipe',
+function isAuthenticated() {
+  const { stdout } = childProcess.spawnSync(reclientHelperPath, ['status'], {
+    cwd: reclientDir,
+    stdio: ['ignore'],
   });
-  if (result.status !== 0) {
-    console.error(result.stdout.toString());
+
+  const match = stdout
+    .toString()
+    .trim()
+    .match(/Authentication Status:\s*(.*)/);
+
+  return match ? match[1].trim() === 'Authenticated' : false;
+}
+
+function auth() {
+  if (getTargetPlatform() === null) {
+    fatal('Unsupported platform for reclient');
+  }
+
+  if (isAuthenticated()) return;
+
+  const { status } = childProcess.spawnSync(reclientHelperPath, ['login'], {
+    stdio: 'inherit',
+  });
+
+  if (status !== 0) {
+    if (process.env.CODESPACES) {
```

Comment:
I'm not sure this is what we want, on code spaces we should leave reclient enabled, just don't try login during init?

Reply:
meaning you'd prefer we maintain that state? i ended up just doing this as I figured it'd be easier